### ISSUE TITLE
Display simulation results

### ISSUE THREAD

**Issue by lochel** [modelica-tools/atom-language-modelica#30]
It would be quite handy if one could open simulation result files right within atom. Is there anyone else who would like to see that?

**Comment by thorade** [comment]
Do you mean open .mat files in Atom? As tables? As diagrams?
Do you also want to start/control simulation from Atom?
Something like this: 
http://www.onewind.de/OneModelica.html

**Comment by lochel** [comment]
I would just like to open .mat files which get displayed as diagrams with a small set of interactive functionality, e.g. selecting of signals that get plotted, zooming and such stuff.

**Comment by thorade** [comment]
This package is also interesting:
https://atom.io/packages/hydrogen
Would it be possible to add Modelica to the supported Jupyter kernels?
https://jupyter.org/

**Comment by bilderbuchi** [comment]
Yeah, a Modelica Jupyter Kernel would be awesome. This has also already been proposed in the [OpenModelica Tracker](https://trac.openmodelica.org/OpenModelica/ticket/3705).

**Comment by thorade** [comment]
@bilderbuchi could you send me an email to m.thorade@udk-berlin.de
